functions.sh

```diff
@@ -1525,15 +1525,25 @@ unmount_all() {
 # TODO: adopt to systemd
 #
 stop_lvm_raid() {
-  test -x /etc/init.d/lvm && /etc/init.d/lvm stop &>/dev/null
-  test -x /etc/init.d/lvm2 && /etc/init.d/lvm2 stop &>/dev/null
+  if systemctl --version &>/dev/null; then
+    for unit in 'mdmonitor' 'lvm2-monitor'; do
+      if grep -q "${unit}\\.service" < <(systemctl list-units); then
+        systemctl stop "${unit}.service"
+      fi
+    done
+  else
+    [[ -x /etc/init.d/lvm ]] && /etc/init.d/lvm stop &>/dev/null
+    [[ -x /etc/init.d/lvm2 ]] && /etc/init.d/lvm2 stop &>/dev/null
+  fi
 
-  dmsetup remove_all > /dev/null 2>&1
+  dmsetup remove_all &>/dev/null
```

Comment:
may be a matter of taste, but I'd go with an extra whitespace`&> /dev/null`

Reply:
As some of the current code base was imported from another project and because we have no bash style guide for this project these stylistic issues are non-relevant at this point.
If you want to, feel free to write a comprehensive style guide, ideally with proper enforcement through a linter and then fix everything in the code. :)